diff --git a/sites/all/modules/contrib/sts_commons_forum/sts_commons_forum.features.inc b/sites/all/modules/contrib/sts_commons_forum/sts_commons_forum.features.inc
index 4bc097e..d7df0bb 100644
--- a/sites/all/modules/contrib/sts_commons_forum/sts_commons_forum.features.inc
+++ b/sites/all/modules/contrib/sts_commons_forum/sts_commons_forum.features.inc
@@ -4,25 +4,17 @@
  * Implementation of hook_ctools_plugin_api().
  */
 function sts_commons_forum_ctools_plugin_api() {
-  module_load_include('inc', 'sts_commons_forum', 'sts_commons_forum.defaults');
-  $args = func_get_args();
-  return call_user_func_array('_sts_commons_forum_ctools_plugin_api', $args);
+  list($module, $api) = func_get_args();
+  if ($module == "strongarm" && $api == "strongarm") {
+    return array("version" => 1);
+  }
 }
 
 /**
- * Implementation of hook_strongarm().
+ * Implementation of hook_views_api().
  */
-function sts_commons_forum_strongarm() {
-  module_load_include('inc', 'sts_commons_forum', 'sts_commons_forum.defaults');
-  $args = func_get_args();
-  return call_user_func_array('_sts_commons_forum_strongarm', $args);
-}
-
-/**
- * Implementation of hook_views_default_views().
- */
-function sts_commons_forum_views_default_views() {
-  module_load_include('inc', 'sts_commons_forum', 'sts_commons_forum.features.views');
-  $args = func_get_args();
-  return call_user_func_array('_sts_commons_forum_views_default_views', $args);
+function sts_commons_forum_views_api() {
+  return array(
+    'api' => '2',
+  );
 }
