diff --git a/profiles/drupal_commons/modules/features/commons_discussion/commons_discussion.features.inc b/profiles/drupal_commons/modules/features/commons_discussion/commons_discussion.features.inc
index 4196e50..26b86ee 100644
--- a/profiles/drupal_commons/modules/features/commons_discussion/commons_discussion.features.inc
+++ b/profiles/drupal_commons/modules/features/commons_discussion/commons_discussion.features.inc
@@ -16,7 +16,7 @@ function commons_discussion_ctools_plugin_api() {
 function commons_discussion_node_info() {
   $items = array(
     'discussion' => array(
-      'name' => t('Discussion'),
+      'name' => t('Dyskusja'),
       'module' => 'features',
       'description' => t('A item which allows the open discussion of a given topic.'),
       'has_title' => '1',
