diff --git a/profiles/drupal_commons/modules/features/commons_discussion/commons_discussion.strongarm.inc b/profiles/drupal_commons/modules/features/commons_discussion/commons_discussion.strongarm.inc
index b62dc89..590c1cb 100644
--- a/profiles/drupal_commons/modules/features/commons_discussion/commons_discussion.strongarm.inc
+++ b/profiles/drupal_commons/modules/features/commons_discussion/commons_discussion.strongarm.inc
@@ -90,6 +90,13 @@ function commons_discussion_strongarm() {
   $strongarm = new stdClass;
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
+  $strongarm->name = 'language_content_type_discussion';
+  $strongarm->value = '0';
+
+  $export['language_content_type_discussion'] = $strongarm;
+  $strongarm = new stdClass;
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
   $strongarm->name = 'node_options_discussion';
   $strongarm->value = array(
     0 => 'status',
@@ -104,5 +111,12 @@ function commons_discussion_strongarm() {
   $strongarm->value = 'group_post_standard';
 
   $export['og_content_type_usage_discussion'] = $strongarm;
+  $strongarm = new stdClass;
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'upload_discussion';
+  $strongarm->value = '0';
+
+  $export['upload_discussion'] = $strongarm;
   return $export;
 }
