diff --git a/profiles/drupal_commons/modules/features/commons_poll/commons_poll.features.menu_links.inc b/profiles/drupal_commons/modules/features/commons_poll/commons_poll.features.menu_links.inc
index e3967b7..b3b447c 100644
--- a/profiles/drupal_commons/modules/features/commons_poll/commons_poll.features.menu_links.inc
+++ b/profiles/drupal_commons/modules/features/commons_poll/commons_poll.features.menu_links.inc
@@ -11,8 +11,13 @@ function commons_poll_menu_default_menu_links() {
     'menu_name' => 'primary-links',
     'link_path' => 'content/polls',
     'router_path' => 'content/polls',
-    'link_title' => 'Polls',
-    'options' => array(),
+    'link_title' => 'Ankiety',
+    'options' => array(
+      'minipanel' => '',
+      'attributes' => array(
+        'title' => '',
+      ),
+    ),
     'module' => 'commons',
     'hidden' => '0',
     'external' => '0',
@@ -23,7 +28,7 @@ function commons_poll_menu_default_menu_links() {
   );
   // Translatables
   // Included for use with string extractors like potx.
-  t('Polls');
+  t('Ankiety');
 
 
   return $menu_links;
