diff --git a/profiles/drupal_commons/modules/features/commons_poll/commons_poll.views_default.inc b/profiles/drupal_commons/modules/features/commons_poll/commons_poll.views_default.inc
index 5bb100b..73399c2 100644
--- a/profiles/drupal_commons/modules/features/commons_poll/commons_poll.views_default.inc
+++ b/profiles/drupal_commons/modules/features/commons_poll/commons_poll.views_default.inc
@@ -487,7 +487,7 @@ function commons_poll_views_default_views() {
   $handler->override_option('cache', array(
     'type' => 'none',
   ));
-  $handler->override_option('title', 'Polls');
+  $handler->override_option('title', 'Ankiety');
   $handler->override_option('header_format', '1');
   $handler->override_option('header_empty', 1);
   $handler->override_option('empty', 'No polls have been added yet.');
@@ -1025,7 +1025,7 @@ function commons_poll_views_default_views() {
     'comments' => 0,
   ));
   $handler = $view->new_display('block', 'Block', 'block_1');
-  $handler->override_option('block_description', 'Latest group poll');
+  $handler->override_option('block_description', 'Najnowsze ankiety');
   $handler->override_option('block_caching', -1);
 
   $views[$view->name] = $view;
@@ -2303,7 +2303,7 @@ function commons_poll_views_default_views() {
   $handler->override_option('cache', array(
     'type' => 'none',
   ));
-  $handler->override_option('title', 'Polls');
+  $handler->override_option('title', 'Ankiety');
   $handler->override_option('header_format', '1');
   $handler->override_option('header_empty', 0);
   $handler->override_option('empty', 'No polls have been added to this group yet.');
