diff --git a/profiles/drupal_commons/modules/features/commons_poll/commons_poll.features.user_permission.inc b/profiles/drupal_commons/modules/features/commons_poll/commons_poll.features.user_permission.inc
index 81fe2ee..e8a2f05 100644
--- a/profiles/drupal_commons/modules/features/commons_poll/commons_poll.features.user_permission.inc
+++ b/profiles/drupal_commons/modules/features/commons_poll/commons_poll.features.user_permission.inc
@@ -10,7 +10,8 @@ function commons_poll_user_default_permissions() {
   $permissions['cancel own vote'] = array(
     'name' => 'cancel own vote',
     'roles' => array(
-      '0' => 'authenticated user',
+      '0' => 'Użytkownik',
+      '1' => 'authenticated user',
     ),
   );
 
@@ -18,7 +19,8 @@ function commons_poll_user_default_permissions() {
   $permissions['create poll content'] = array(
     'name' => 'create poll content',
     'roles' => array(
-      '0' => 'authenticated user',
+      '0' => 'Użytkownik',
+      '1' => 'authenticated user',
     ),
   );
 
@@ -34,7 +36,8 @@ function commons_poll_user_default_permissions() {
   $permissions['delete own poll content'] = array(
     'name' => 'delete own poll content',
     'roles' => array(
-      '0' => 'authenticated user',
+      '0' => 'Użytkownik',
+      '1' => 'authenticated user',
     ),
   );
 
@@ -50,23 +53,23 @@ function commons_poll_user_default_permissions() {
   $permissions['edit own poll content'] = array(
     'name' => 'edit own poll content',
     'roles' => array(
-      '0' => 'authenticated user',
+      '0' => 'Użytkownik',
+      '1' => 'authenticated user',
     ),
   );
 
   // Exported permission: inspect all votes
   $permissions['inspect all votes'] = array(
     'name' => 'inspect all votes',
-    'roles' => array(
-      '0' => 'authenticated user',
-    ),
+    'roles' => array(),
   );
 
   // Exported permission: vote on polls
   $permissions['vote on polls'] = array(
     'name' => 'vote on polls',
     'roles' => array(
-      '0' => 'authenticated user',
+      '0' => 'Użytkownik',
+      '1' => 'authenticated user',
     ),
   );
 
