diff --git a/profiles/drupal_commons/modules/features/commons_reputation/commons_reputation.info b/profiles/drupal_commons/modules/features/commons_reputation/commons_reputation.info
index 9bc5cb6..7042a8d 100644
--- a/profiles/drupal_commons/modules/features/commons_reputation/commons_reputation.info
+++ b/profiles/drupal_commons/modules/features/commons_reputation/commons_reputation.info
@@ -2,6 +2,7 @@ core = "6.x"
 dependencies[] = "commons_core"
 dependencies[] = "context"
 dependencies[] = "features"
+dependencies[] = "location"
 dependencies[] = "menu"
 dependencies[] = "shoutbox_points"
 dependencies[] = "strongarm"
@@ -36,7 +37,6 @@ features[variable][] = "upload_group"
 features[variable][] = "upload_notice"
 features[variable][] = "upload_page"
 features[variable][] = "upload_poll"
-features[variable][] = "upload_wiki"
 features[variable][] = "user_badges_showone"
 features[variable][] = "userpoints_post__undo_points_on_delete"
 features[variable][] = "userpoints_post_blog"
