diff --git a/profiles/drupal_commons/modules/features/commons_wiki/commons_wiki.strongarm.inc b/profiles/drupal_commons/modules/features/commons_wiki/commons_wiki.strongarm.inc
index 791d998..ff0a4f4 100644
--- a/profiles/drupal_commons/modules/features/commons_wiki/commons_wiki.strongarm.inc
+++ b/profiles/drupal_commons/modules/features/commons_wiki/commons_wiki.strongarm.inc
@@ -127,6 +127,13 @@ function commons_wiki_strongarm() {
   $strongarm = new stdClass;
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
+  $strongarm->name = 'language_content_type_wiki';
+  $strongarm->value = '0';
+
+  $export['language_content_type_wiki'] = $strongarm;
+  $strongarm = new stdClass;
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
   $strongarm->name = 'node_options_wiki';
   $strongarm->value = array(
     0 => 'status',
@@ -145,6 +152,13 @@ function commons_wiki_strongarm() {
   $strongarm = new stdClass;
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
+  $strongarm->name = 'upload_wiki';
+  $strongarm->value = '0';
+
+  $export['upload_wiki'] = $strongarm;
+  $strongarm = new stdClass;
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
   $strongarm->name = 'wikitools_404_type';
   $strongarm->value = array(
     'Link to creation' => 'Link to creation',
