diff --git a/profiles/drupal_commons/modules/features/commons_wiki/commons_wiki.info b/profiles/drupal_commons/modules/features/commons_wiki/commons_wiki.info
index fab8c98..38cc46f 100644
--- a/profiles/drupal_commons/modules/features/commons_wiki/commons_wiki.info
+++ b/profiles/drupal_commons/modules/features/commons_wiki/commons_wiki.info
@@ -7,6 +7,7 @@ dependencies[] = "features"
 dependencies[] = "fieldgroup"
 dependencies[] = "freelinking"
 dependencies[] = "imagefield"
+dependencies[] = "location"
 dependencies[] = "menu"
 dependencies[] = "og_views"
 dependencies[] = "optionwidgets"
@@ -47,8 +48,10 @@ features[variable][] = "freelinking_nodetype"
 features[variable][] = "freelinking_notfound"
 features[variable][] = "freelinking_onceonly"
 features[variable][] = "freelinking_restriction"
+features[variable][] = "language_content_type_wiki"
 features[variable][] = "node_options_wiki"
 features[variable][] = "og_content_type_usage_wiki"
+features[variable][] = "upload_wiki"
 features[variable][] = "wikitools_404_type"
 features[variable][] = "wikitools_disallowed_characters"
 features[variable][] = "wikitools_main_page_title"
