diff --git a/profiles/drupal_commons/modules/features/commons_document/commons_document.info b/profiles/drupal_commons/modules/features/commons_document/commons_document.info
index 303454d..98fe8ab 100644
--- a/profiles/drupal_commons/modules/features/commons_document/commons_document.info
+++ b/profiles/drupal_commons/modules/features/commons_document/commons_document.info
@@ -8,6 +8,7 @@ dependencies[] = "fieldgroup"
 dependencies[] = "filefield"
 dependencies[] = "filefield_meta"
 dependencies[] = "getid3"
+dependencies[] = "location"
 dependencies[] = "menu"
 dependencies[] = "og_views"
 dependencies[] = "optionwidgets"
@@ -47,8 +48,10 @@ features[variable][] = "comment_form_location_document"
 features[variable][] = "comment_preview_document"
 features[variable][] = "comment_subject_field_document"
 features[variable][] = "content_extra_weights_document"
+features[variable][] = "language_content_type_document"
 features[variable][] = "node_options_document"
 features[variable][] = "og_content_type_usage_document"
+features[variable][] = "upload_document"
 features[views][] = "content_document_page"
 features[views][] = "document_latest_comments"
 features[views][] = "group_block_doc_latest_comments"
