diff --git a/profiles/drupal_commons/modules/features/commons_document/commons_document.features.menu_links.inc b/profiles/drupal_commons/modules/features/commons_document/commons_document.features.menu_links.inc
index 7e27e5a..ddbfb91 100644
--- a/profiles/drupal_commons/modules/features/commons_document/commons_document.features.menu_links.inc
+++ b/profiles/drupal_commons/modules/features/commons_document/commons_document.features.menu_links.inc
@@ -11,8 +11,13 @@ function commons_document_menu_default_menu_links() {
     'menu_name' => 'primary-links',
     'link_path' => 'content/documents',
     'router_path' => 'content/documents',
-    'link_title' => 'Documents',
-    'options' => array(),
+    'link_title' => 'Dokumenty',
+    'options' => array(
+      'minipanel' => '',
+      'attributes' => array(
+        'title' => '',
+      ),
+    ),
     'module' => 'commons',
     'hidden' => '0',
     'external' => '0',
@@ -23,7 +28,7 @@ function commons_document_menu_default_menu_links() {
   );
   // Translatables
   // Included for use with string extractors like potx.
-  t('Documents');
+  t('Dokumenty');
 
 
   return $menu_links;
