diff --git a/profiles/drupal_commons/modules/features/commons_dashboard/commons_dashboard.info b/profiles/drupal_commons/modules/features/commons_dashboard/commons_dashboard.info
index c360d97..d066142 100644
--- a/profiles/drupal_commons/modules/features/commons_dashboard/commons_dashboard.info
+++ b/profiles/drupal_commons/modules/features/commons_dashboard/commons_dashboard.info
@@ -4,6 +4,7 @@ dependencies[] = "content"
 dependencies[] = "features"
 dependencies[] = "homebox"
 dependencies[] = "jquery_ui"
+dependencies[] = "location"
 dependencies[] = "menu"
 dependencies[] = "og_views"
 dependencies[] = "views"
