diff --git a/profiles/drupal_commons/modules/features/commons_dashboard/commons_dashboard.views_default.inc b/profiles/drupal_commons/modules/features/commons_dashboard/commons_dashboard.views_default.inc
index 0a0688f..91bce9e 100644
--- a/profiles/drupal_commons/modules/features/commons_dashboard/commons_dashboard.views_default.inc
+++ b/profiles/drupal_commons/modules/features/commons_dashboard/commons_dashboard.views_default.inc
@@ -242,7 +242,7 @@ function commons_dashboard_views_default_views() {
       'relationship' => 'none',
     ),
     'title' => array(
-      'label' => 'Name',
+      'label' => 'Nazwa',
       'alter' => array(
         'alter_text' => 0,
         'text' => '',
@@ -258,8 +258,8 @@ function commons_dashboard_views_default_views() {
         'max_length' => '',
         'word_boundary' => 1,
         'ellipsis' => 1,
-        'strip_tags' => 0,
         'html' => 0,
+        'strip_tags' => 0,
       ),
       'empty' => '',
       'hide_empty' => 0,
@@ -270,9 +270,12 @@ function commons_dashboard_views_default_views() {
       'table' => 'node',
       'field' => 'title',
       'relationship' => 'none',
+      'override' => array(
+        'button' => 'Use default',
+      ),
     ),
     'post_count_new' => array(
-      'label' => 'New posts',
+      'label' => 'Nowe posty',
       'alter' => array(
         'alter_text' => 0,
         'text' => '',
@@ -310,7 +313,7 @@ function commons_dashboard_views_default_views() {
       'relationship' => 'none',
     ),
   ));
-  $handler->override_option('title', 'My groups');
+  $handler->override_option('title', 'Moje grupy');
   $handler->override_option('empty', 'You haven\'t joined any groups yet.');
   $handler->override_option('empty_format', '1');
   $handler->override_option('use_pager', 'mini');
