diff --git a/profiles/drupal_commons/modules/features/commons_core/commons_core.context.inc b/profiles/drupal_commons/modules/features/commons_core/commons_core.context.inc
index 51772c3..f46e153 100644
--- a/profiles/drupal_commons/modules/features/commons_core/commons_core.context.inc
+++ b/profiles/drupal_commons/modules/features/commons_core/commons_core.context.inc
@@ -64,15 +64,15 @@ function commons_core_context_default_contexts() {
         ),
         'commons_core-3' => array(
           'module' => 'commons_core',
-          'delta' => '3',
+          'delta' => 3,
           'region' => 'header',
           'weight' => 1,
         ),
-        'commons_core-2' => array(
-          'module' => 'commons_core',
-          'delta' => '2',
-          'region' => 'footer',
-          'weight' => 0,
+        'privatemsg-privatemsg-new' => array(
+          'module' => 'privatemsg',
+          'delta' => 'privatemsg-new',
+          'region' => 'header',
+          'weight' => 2,
         ),
       ),
     ),
@@ -116,17 +116,23 @@ function commons_core_context_default_contexts() {
           'region' => 'sidebar_last',
           'weight' => 1,
         ),
+        'og_subgroups-0' => array(
+          'module' => 'og_subgroups',
+          'delta' => 0,
+          'region' => 'sidebar_last',
+          'weight' => 2,
+        ),
         'views-og_members_custom_block-block_1' => array(
           'module' => 'views',
           'delta' => 'og_members_custom_block-block_1',
           'region' => 'sidebar_last',
-          'weight' => 2,
+          'weight' => 3,
         ),
         'views-og_tag_cloud-block_1' => array(
           'module' => 'views',
           'delta' => 'og_tag_cloud-block_1',
           'region' => 'sidebar_last',
-          'weight' => 3,
+          'weight' => 4,
         ),
       ),
     ),
@@ -170,6 +176,12 @@ function commons_core_context_default_contexts() {
           'region' => 'sidebar_last',
           'weight' => 1,
         ),
+        'views-profile_friends-block_1' => array(
+          'module' => 'views',
+          'delta' => 'profile_friends-block_1',
+          'region' => 'sidebar_last',
+          'weight' => 2,
+        ),
       ),
     ),
   );
